BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -333,11 +333,20 @@ GetContainerKeyUsageBySig (
   IN  UINT32    ContainerSig
   )
 {
+  UINT8  Idx;
+
   if (ContainerSig == CONTAINER_BOOT_SIGNATURE) {
     return HASH_USAGE_PUBKEY_OS;
-  } else {
-    return HASH_USAGE_PUBKEY_CONTAINER_DEF;
   }
+
+  if ((ContainerSig & 0x00FFFFFF) == CONTAINER_OEM_BASE_SIGNATURE) {

```

Comment:
Does this condition works when container uses signature as 
"SIGNATURE_32 ('O', 'E', 'M',   **1)"**?

Reply:
Container signature is expected to be 'OEM0' to 'OEM7'.  We will match 'OEM' part first, then decode the 0-7 number.   SIGNATURE_32 ('O', 'E', 'M', 1) will not be considered as a valid OEM signture.  it needs to be SIGNATURE_32 ('O', 'E', 'M', '1').

Reply:
Thanks. I overlooked 0x**00**FFFFFF.